l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -479,18 +515,20 @@ def _CreatePayamentsLine(
                     )
                     if not bankids:
                         if dline.IstitutoFinanziario == '':
```

Comment:
dline.IstitutoFinanziario potrebbe essere None, giusto? In tal caso verrebbe cmq eseguita la riga 527
Suggerisco ```if not dline.IstitutoFinanziario```